types/node/index.d.ts

```diff
@@ -797,82 +809,93 @@ declare module "http" {
     /**
      * @deprecated Use IncomingMessage
      */
-    export interface ServerRequest extends IncomingMessage {
+    export class ServerRequest extends IncomingMessage {
         connection: net.Socket;
     }
 
-    export interface ServerResponseHeaders {
-      [key: string]: number | string | string[];
-    }
+    // https://github.com/nodejs/node/blob/master/lib/_http_outgoing.js
+    export class OutgoingMessage extends stream.Writable {
+        output: any[];
```

Comment:
since `OutgoingMessage` is not documented on Nodejs official document, I had hard time determine which should be kept in props.
I think we should keep props below for sure.
``` ts
        upgrading: boolean;
        chunkedEncoding: boolean;
        shouldKeepAlive: boolean;
        useChunkedEncodingByDefault: boolean;
        sendDate: boolean;
        finished: boolean;
        headersSent: boolean;
        connection: net.Socket;
```

and remove `output`, `outputEncodings`, `outputCallbacks`.
@Flarna what to you think?